lib/medium-admin.php

```diff
@@ -136,10 +136,16 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters('medium_allowed_post_types', array('post'));
```

Comment:
@majelbstoat You would think so, but actually if you publish any Page it gets pushed to Medium automatically. Also mayhem occurs when saving nav menu items. I think whitelisting post types is the right way to go to solve these issues.

<img width="976" alt="screen shot 2015-10-08 at 8 05 39 pm" src="https://cloud.githubusercontent.com/assets/522158/10383710/1f496bd4-6df8-11e5-83fe-2601676c4bb3.png">